content/en/blog/_posts/2021-08-11-memory-qos-cgroups-v2.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: 'Support memory qos with cgroups v2'
+date: 2021-08-11
+slug: memory-qos-cgroups-v2
+---
+
+**Authors:** Tim Xu (Tencent Cloud)
+
+Memory QoS is used to memory protection and usage throttle on pod / container to increase memory isolation and service quality.
```

Comment:
```suggestion
Memory QoS provides memory protection and usage throttling on pods and containers to increase memory isolation and service quality.
```

Reply:
```suggestion
Kubernetes v1.22 adds a new (alpha) feature that improves how Linux nodes
implement memory resource requests and limits. The new feature relies on
cgroups v2; most current operating system releases for Linux already provide that.
With this experimental feature,
[quality-of-service for pods and containers](/docs/tasks/configure-pod-container/quality-service-pod/)
extends to cover not just CPU time but RAM as well.
```

(I think it's useful to link to that page even though it has been overlooked for an update about the new Memory QoS feature: we can fix the task page post-release, and all the time it's still the best place for the announcement to link to).